test/sharness/t0066-migration.sh

```diff
@@ -10,16 +10,27 @@ test_description="Test migrations auto update prompt"
 
 test_init_ipfs
 
+# Create fake migration binaries instead of letting ipfs download from network
+# To test downloading and running actual binaries, comment out this test.
```

Comment:
We weren't testing real binaries here before, right? So this is just a heads up for developers/testers.

Reply:
The actual migrations are tested in fs-repo-migrations sharness tests.  This part tests that the go-ipfs will actually run the migrations.

The part that is missing from a complete end-to-end test is testing go-ipfs downloading migrations from the network.